app/assets/stylesheets/common/topic-timeline.scss

```diff
@@ -251,6 +251,7 @@
       float: left;
       z-index: z("base") + 1;
       outline: 1px solid transparent;
+      margin-left: 0.5px;
```

Comment:
Is this going to have unintended consequences for the non-glimmer topic timeline?

Reply:
No, in the widget version it is off slightly as well

# Before
<img width="324" alt="Screenshot 2023-02-08 at 11 41 38 AM" src="https://user-images.githubusercontent.com/50783505/217609423-36c65d6e-e7d5-41d5-9577-3890623c8b36.png">

# After
<img width="317" alt="Screenshot 2023-02-08 at 11 41 51 AM" src="https://user-images.githubusercontent.com/50783505/217609473-1eb8f4cb-b347-4168-9ae4-78e0c2823a20.png">